### PR TITLE
Feature surinder

### PR DESCRIPTION
**Pull request by Surinder07**
I added some features in html file, I added js file

### CODE REVIEW

**Comment by 12shack** [comment]
this pull request is good and ready for request.

**Comment by SChaudhary99** [comment]
this file is rready to merge

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <title>Page Title</title>
 </head>
 <body>
-
+<h2>added some lines in feature branch</h2>
```
Comment (mthakurwani) [inline index.html:7]: this pull request is good

**Review by mthakurwani — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Farzaneh-ra) [inline index.html:7]: test!

**Review by Farzaneh-ra — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 <title>Page Title</title>
 </head>
 <body>
-
```
Comment (SChaudhary99) [inline index.html:7]: this looks good

**Pull request merged**